python/servo/bootstrap_commands.py

```diff
@@ -63,6 +63,9 @@ def download(desc, src, writer, start_byte=0):
 
         if not dumb:
             print()
+    except urllib2.URLError:
+        print("Error downloading Rust compiler; are you connected to the internet?")
```

Comment:
Okay, I now realize that there's already a message in a similar format. We're good then :)